cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskTracker.java

```diff
@@ -18,8 +18,10 @@
 public class ExecutionTaskTracker {
   // Dead tasks indicate cancelled/killed executions affecting the original state before the balancing action execution.
   private final Map<BalancingAction, Set<ExecutionTask>> _deadTasks;
-  // Aborted tasks indicate tasks stopped due to (1) cancelled balancing actions not affecting the state before the
-  // execution of balancing action or (2) deletion of topic partitions for which the balancing action was in progress.
+  // Aborting tasks due to (1) cancelled balancing actions not affecting the state before the execution of balancing 
+  // action.
+  private final Map<BalancingAction, Set<ExecutionTask>> _abortingTasks;
+  // Tasks that have been successfully aborted.
```

Comment:
What does successfully aborted tasks involve? Does deletion of topic partitions for which the balancing action was in progress yield aborted tasks?

Reply:
Yes, it is defined as aborted. i.e. no impact was left.